packages/java/tests/spring/react-grid-test/frontend/themes/react-grid-test/styles.css

```diff
@@ -1 +1,2 @@
-/* Import your application global css files here or add the styles directly to this file */
\ No newline at end of file
+/* Import your application global css files here or add the styles directly to this file */
+@import url('@vaadin/vaadin-lumo-styles/lumo.css');
```

Comment:
The component's custom CSS uses Lumo variables so for now they only work properly with the Lumo theme.